### PR TITLE
4 simple ui

### PR DESCRIPTION
**Pull request by runalmaas**

### CODE REVIEW

File: src/main/java/no/uib/inf112/core/RoboRally.java [hunk 1]
```diff
@@ -34,6 +36,10 @@
     private Robot robot;
     private Robot robot2;
 
+    InputMultiplexer inputMultiplexer;
+
+    UIHandler uiHandler;
```
Comment (kh498) [inline src/main/java/no/uib/inf112/core/RoboRally.java]: Make these private

**Comment by kh498** [comment]
Also please merge master into your branch

**Comment by kh498** [comment]
As of of now this is really for proof of concept for interaction @DanielBerge I don't think UIHandler needs this yet

**Comment by runalmaas** [comment]
Fixed

**Comment by DanielBerge** [comment]
Fixes #4

**Pull request merged**